src/Data/API/BaseApiHelper.ts

```diff
@@ -33,26 +35,32 @@ export class BaseApiHelper implements ApiHelper {
 
   private getHeaders(environment?: string): Record<string, string> {
     const { keycloak } = this;
+    const jwt = getCookie("inmanta_user");
```

Comment:
Ad1) This cookie set only when user use database login, now as I think about it ,the only question is, whether it is possible that end-user toggle between authorisations? @bartv 

Ad2) I'll extract it

Reply:
It is not possible to switch. However, just like with keycloak we need to always pass along the token.

Reply:
I'd still appreciate if it was extracted. I think it makes it clearer and even if theoretically it would never end up in the loophole, it's always best to avoid creating them when we can.

Reply:
extracted